python/llm/test/inference/test_transformers_api.py

```diff
@@ -49,16 +49,16 @@ def test_transformers_auto_model_int4(self):
         print('Prompt:', input_str)
         print('Output:', output_str)
         print(f'Inference time: {end-st} s')
-        res = 'Paris' in output_str        
+        res = 'Paris' in output_str
         self.assertTrue(res)
 
     def test_transformers_auto_model_for_causal_lm_int4(self):
-        model_path = os.environ.get('ORIGINAL_REPLIT_CODE_PATH')
+        model_path = os.environ.get('ORIGINAL_CODESHELL_7B_PATH')
```

Comment:
What is the reason for this change?

Reply:
replit-code-v1-3b cannot run with transformers 4.36, so another code generation model is used here instead :)